crates/iota-node/src/lib.rs

```diff
@@ -2067,16 +2066,9 @@ pub async fn build_http_server(
             ))?;
         }
 
-        let iota_names_config = match &config.iota_names_config {
-            Some(config) => config.clone(),
-            None => {
-                let chain = state
-                    .get_chain_identifier()
-                    .ok_or(IotaError::Storage("chain identifier not found".to_string()))?
-                    .chain();
-                IotaNamesConfig::from_chain(&chain)
-            }
-        };
+        // TODO: Init from chain if config is not set when `IotaNamesConfig::from_chain`
+        // once implemented
```

Comment:
Is there an issue with the existing code or why do you want to change it?

Reply:
Yes, the existing `IotaNamesConfig::from_chain` code is not fully implemeted / has `todo`s. As soon as the software is released on devnet/testnet/mainnet while iota-names is not fully activated, it would end up in a panic.